playground/pspin/pspin.py

```diff
@@ -0,0 +1,62 @@
+import matplotlib
```

Comment:
ideally you want to put this whole file into a main() function.  I've run into annoying problems with global variables by failing to do that.  It's also just good habit.